pkgs/development/python-modules/mkdocs-git-committers-plugin-2/default.nix

```
@@ -0,0 +1,43 @@
+{
+  # Evaluation
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonAtLeast,
+
+  # Dependencies
+  gitpython,
+  mkdocs,
+  requests,
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-git-committers-plugin-2";
+  version = "2.4.1";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
and please set `build-system` to avoid the legacy build.
see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters